itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCompactorBase.java

```diff
@@ -89,6 +89,7 @@ public void setup() throws Exception {
     hiveConf.setBoolVar(HiveConf.ConfVars.HIVEOPTIMIZEMETADATAQUERIES, false);
     MetastoreConf.setBoolVar(hiveConf, MetastoreConf.ConfVars.COMPACTOR_INITIATOR_ON, true);
     MetastoreConf.setBoolVar(hiveConf, MetastoreConf.ConfVars.COMPACTOR_CLEANER_ON, true);
+    MetastoreConf.setBoolVar(hiveConf, MetastoreConf.ConfVars.COMPACTOR_CLEAN_ABORTS_USING_CLEANER, false);
```

Comment:
why do we need this config? let's keep it simple

Reply:
The plan is to keep this config till some point until we know that abort cleanup is stable. If there are any issues with the given handler, we can use the compaction cycle. I will create a task once its determined that this feature is stable and we can remove this feature flag then.